### PR TITLE
Refactored commands into exported api

### PR DESCRIPTION
**Pull request by elmasse**
Can you guys check if this PR won't break anything? Maybe it can be released as 1.3.0-alpha to avoid issues and keep working on documentation for api

### CODE REVIEW

File: src/api/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export { defautl as init } from './init';
```
Comment (dpaez) [inline src/api/index.js]: There is a typo. _defautl_ should be __default__.

File: src/cli/commands/init.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
-const initTemplate = `
-# Change Log
-All notable changes to this project will be documented in this file.
+// const initTemplate = `
```
Comment (dpaez) [inline src/cli/commands/init.js]: Commented out lines can be removed now.

File: src/api/index.js (hunk 1 above)
Comment (dpaez) [inline src/api/index.js]: there is a typo. _defautl_ should be **default**

File: src/cli/commands/init.js (hunk 2 above)
Comment (dpaez) [inline src/cli/commands/init.js]: Commented out lines can be removed

File: src/cli/lib/create-command.js [hunk 3]
```diff
@@ -39,18 +39,7 @@ export default function createCommand(cli, userDef = {}) {
...
-                    // write callback function
-                    if (argv.stdout) {
-                        process.stdout.write(data);
-                    } else {
-                        parserInstance.write(data);
-                    }
-                    resolve(data);
-                });
-            }
+            writer({ parserInstance, stdout: argv.stdout })
```
Comment (dpaez) [inline src/cli/lib/create-command.js]: This is :cool:

File: src/api/index.js (hunk 1 above)
Comment (tinchoz49) [inline src/api/index.js]: that was from an old commit

**Pull request merged**